rotkehlchen/db/utils.py

```diff
@@ -151,27 +151,27 @@ class DBAssetBalance(NamedTuple):
     category: BalanceType
     time: Timestamp
     asset: Asset
-    amount: str
-    usd_value: str
+    amount: FVal
```

Comment:
you can also turn this into a data class if you need it at any point to be mutable.